app/subcommands/service_subcommand.py

```diff
@@ -16,6 +17,12 @@ class ServiceSubcommand(Subcommand):
     _THREAD_NAME = None
 
     def run(self, *args, **kwargs):
+        # Set the multiprocessing method to 'spawn'. (The default on unix is 'fork'.) According to documentation, this
```

Comment:
This path will definitely only execute once -- this is pretty early on in the application lifecycle. Our `main.py` calls `run()` directly.

I believe this will raise an exception if called more than once, so functional tests would definitely catch it, and also some unit tests would fail if this wasn't mocked out.